infra/docker-compose.yml

```diff
@@ -31,8 +31,15 @@ services:
       GOTRUE_SMTP_PASS: GOTRUE_SMTP_PASS
       GOTRUE_SMTP_ADMIN_EMAIL: admin@email.com
       GOTRUE_MAILER_SUBJECTS_CONFIRMATION: 'Please confirm'
+      GOTRUE_EXTERNAL_PHONE_ENABLED: 'true'
```

Comment:
just to check if we're on the same page, we don't set `GOTRUE_EXTERNAL_EMAIL_ENABLED='true'` because gotrue sets it to true by default

Reply:
yea 👍